client_grpc.go

```diff
@@ -452,7 +446,36 @@ func (c *clientGrpc) OnlineQueryBulk(args OnlineQueryParamsComplete) (OnlineQuer
 }
 
 func (c *clientGrpc) UploadFeatures(args UploadFeaturesParams) (UploadFeaturesResult, error) {
-	return UploadFeaturesResult{}, errors.New("not implemented")
+	inputsConverted, err := args.getConvertedInputsMap()
+	if err != nil {
+		return UploadFeaturesResult{}, wrapClientError(err, "error converting inputs map")
+	}
+	inputsFeather, err := internal.InputsToArrowBytes(inputsConverted)
+	if err != nil {
+		return UploadFeaturesResult{}, wrapClientError(err, "error serializing inputs as feather")
+	}
+
+	req := connect.NewRequest(&commonv1.UploadFeaturesBulkRequest{
+		InputsFeather: inputsFeather,
+		BodyType:      commonv1.FeatherBodyType_FEATHER_BODY_TYPE_TABLE,
+	})
+
+	res, err := c.queryClient.UploadFeaturesBulk(context.Background(), req)
```

Comment:
is this the right context to use here? Should the caller be able to pass one in? Not sure what is the common thing to do in golang

Reply:
good question - idk either let me go look around

Reply:
addressed